packages/core/src/renderer/logger/renderer-log-file-id.injectable.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import windowLocationInjectable from "../../common/k8s-api/window-location.injectable";
+import currentlyInClusterFrameInjectable from "../routes/currently-in-cluster-frame.injectable";
+import { getClusterIdFromHost } from "../../common/utils";
+
+const rendererLogFileIdInjectable = getInjectable({
+  id: "renderer-log-file-id",
+  instantiate: (di) => {
+    let frameId: string;
+    const currentlyInClusterFrame = di.inject(currentlyInClusterFrameInjectable);
+
+    if (currentlyInClusterFrame) {
+      const { host } = di.inject(windowLocationInjectable);
+      const clusterId = getClusterIdFromHost(host);
+
+      frameId = clusterId ? `cluster-${clusterId}` : "cluster";
```

Comment:
Given that a `clusterId` should always be present, maybe just trying to compute the `getClusterIdFromHost` is enough to verify that we are currently in the cluster frame

Reply:
I removed the ternary condition as it was quite unnecessary. Not sure if you meant `currentlyInClusterFrame` should be replaced with `getClusterIdFromHost`. To me it seems more correct to use `currentlyInClusterFrame` which checks the process frame, getting part of the url host with `getClusterIdFromHost` might accidentally return something for the main frame too.